Client/Ecosia/UI/EcosiaFindInPageBar.swift

```diff
@@ -0,0 +1,238 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import UIKit
+import Shared
+
+protocol EcosiaFindInPageBarDelegate: AnyObject {
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didTextChange text: String)
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didFindPreviousWithText text: String)
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didFindNextWithText text: String)
+    func findInPageDidPressClose(_ findInPage: EcosiaFindInPageBar)
+}
+
+/// Ecosia's custom UI for FindInPageBar.
+///
+/// You can find the Firefox original view in Client/Frontend/Browser/FindInPageBar (removed from Target since no longer used)
+class EcosiaFindInPageBar: UIView {
```

Comment:
Reminder to myself too as a good practice to double-check whether our classes can be made `final` so that we are constantly improving our codebase along.

Reply:
That's right, will update it.